docs/containers-storage.conf.5.md

```diff
@@ -129,7 +129,31 @@ Specifies the maximum number of retries XFS should attempt to complete IO when E
   Tell storage drivers to use the specified OSTree repository.  Some storage drivers, such as overlay, might use
 
 **skip_mount_home=""**
-  Tell storage drivers to not create a PRIVATE bind mount on their home directory.
+p  Tell storage drivers to not create a PRIVATE bind mount on their home directory.
+
+## SElinux labeling.
+
+When running on an SELinux system, if you move the containers storage graphroot directory, you must make sure the labeling is correct.
+
```

Comment:
I think this is done mostly for rootless containers?  If so, you might want to add a little follow on sentence about that.  "This is  especially useful when running a rootless container."  or some such.

Reply:
I don't agree that this done mostly for rootless containers.  The cases I see have been around people not having enough space in /var.  I will be extending this to a blog, to further my ideas on this.  But I don't think we need to mention rootless.